adapters/handlers/grpc/v1/prepare_reply.go

```diff
@@ -56,7 +56,7 @@ func searchResultsToProto(res []interface{}, start time.Time, searchParams dto.G
 	return out, nil
 }
 
-func extractObjectsToResults(res []interface{}, searchParams dto.GetParams, scheme schema.Schema, fromGroup bool) ([]*pb.SearchResult, string, error) {
+func extractObjectsToResults(res []interface{}, searchParams dto.GetParams, scheme schema.Schema, fromGroup, UsesPropertiesMessage bool) ([]*pb.SearchResult, string, error) {
```

Comment:
here also, can we change the variable name so that it start with lowercase letter?